functions/Select-DbaDbSequenceNextValue.ps1

```diff
@@ -0,0 +1,89 @@
+function Select-DbaDbSequenceNextValue {
```

Comment:
I see no reason for this to be a command by itself. That content can be added to the object for `Get-DbaSequence` and simply output the additional property as a default output.

Reply:
Would there be a new param for Get-DbaDbSequence such as -SelectNextValue so that the caller could advance the sequence and obtain the next value? I would be concerned about the Get-DbaDbSequence function changing state on the server (i.e. advancing the sequence). The goal for this Select command is to provide a scripting command to call "select next value for <sequence>".